src/VCEL.Core/Lang/VCELVisitor.cs

```diff
@@ -155,6 +155,12 @@ public override ParseResult<T> VisitInOp([NotNull] VCELParser.InOpContext contex
             });
         }
 
+        public override ParseResult<T> VisitVariable([NotNull] VCELParser.VariableContext context)
```

Comment:
main change here, we have defined the grammar and expr already, but never implement this to be able to parse the expression with "#ID"